uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-calendar.html

```diff
@@ -103,27 +108,45 @@
             </select>
             <span class="crm_custom-select__arrow"></span>
           </div>
-
           <i class="fa fa-question-circle chr_leave-calendar__admin-filters-hint text-info pointer"
              aria-hidden="true"
              ng-click="calendar.showAdminFilteringHint()"></i>
         </div>
       </div>
     </div>
-    <crm-loading show="!calendar.loading.calendar">
-      <div ng-if="calendar.injectMonths">
-        <leave-calendar-month ng-repeat="month in calendar.months track by $index"
+    <section class="panel panel-default">
+      <header class="panel-heading chr_leave-calendar__month-header">
+        <h2 class="panel-title">
+          <button class="btn btn-secondary-outline chr_leave-calendar__month-header__go-to-current-month"
+            ng-click="calendar.selectCurrentMonth()">Today</button>
+          <button class="btn btn-link chr_leave-calendar__month-header__month-paginator"
+            ng-class="{ disabled: !calendar.monthPaginatorsAvailability.previous }"
```

Comment:
you don't need to add a class for this, `button[disabled]` should be enough.

Reply:
Done.